local/demo.py

```diff
@@ -13,8 +15,7 @@ def vm(start, end):
 
 
 # Mock Server
-_API_URL = "http://apiserver:8000/"  # apiserver is for docker-compose. Change this to http://localhost:8000/ for local testing
-# TODO make http://apiserver:8000/ env var for controller and runners and default _API_URL to localhost
+_API_URL = os.environ.get("API_URL", "http://localhost:8000/")
```

Comment:
I think this gets us what I had in mind when I wrote that TODO comment

But another idea (can be something for a future PR or I can add it in, idm):

- We use the mite config manager by setting `MITE_CONF_api_url=http://localhost:8000/` in the docker-compose for the controller and runners image
- `_API_URL = "http://localhost:8000/" for local testing. Tbh we probably don't need this but why not and leave the demo_req as it is
- Make another journey `demo_config_manager_req`

which would look like 
```python
@mite_http
async def demo_config_manager_req(ctx):
    base_url = ctx.config.get('MITE_CONF_api_url')
    async with ctx.transaction("Get request"):
        await ctx.http.get(base_url)
```